can_bus/quadruna/BMS/BMS_alerts.json

```diff
@@ -0,0 +1,38 @@
+{
+    "warnings_id": 150,
+    "faults_id": 151,
+    "warnings_counts_id": 152,
+    "faults_counts_id": 153,
+    "warnings": [
+        "StackWaterMarkHighTask1Hz",
+        "StackWaterMarkHighTask100Hz",
+        "StackWaterMarkHighTask1kHz",
+        "StackWaterMarkHighTaskCanRx",
+        "StackWaterMarkHighTaskCanTx",
+        "WatchdogTimeout",
+        "TxOverflow",
+        "RxOverflow"
+    ],
+    "faults": [
+        "MissingFSMHeartbeat",
+        "MissingDCMHeartbeat",
+        "MissingPDMHeartbeat",
```

Comment:
Board names have changed

Reply:
Will update this when HeartbeatBoards enum is updated, as will need to leave other sections of code as TODO until then anyways